<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaSecundaria</class>
 <widget class="QWidget" name="ventanaSecundaria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_buscarNumero">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingresa un numero..</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_buscarElemento">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Buscar Elemento</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ubicacion">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ubicacion</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_cantVeces">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cantidad de veces que se repite</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
